pkg/manifests/manifests.go

```diff
@@ -876,6 +876,21 @@ func (f *Factory) updateNodeExporterArgs(args []string) ([]string, error) {
 		args = setArg(args, "--no-collector.tcpstat", "")
 	}
 
+	if f.config.ClusterMonitoringConfiguration.NodeExporterConfig.Collectors.Sysctl.Enabled {
+		includeSysctlMetrics := *f.config.ClusterMonitoringConfiguration.NodeExporterConfig.Collectors.Sysctl.IncludeSysctlMetrics
+		includeInfoSysctlMetrics := *f.config.ClusterMonitoringConfiguration.NodeExporterConfig.Collectors.Sysctl.IncludeInfoSysctlMetrics
+
+		args = setArg(args, "--collector.sysctl", "")
+		for _, sysctl := range includeSysctlMetrics {
```

Comment:
we need to exclude duplicate values otherwise the collector will fail. It could be an improvement for upstream btw.